### PR TITLE
code improvements

### PR DESCRIPTION
**Pull request by AlexJuca**
This PR contains the following changes and improvements:

- Cleans up documentation, artifacts of a previous conflict were lingering in the docs.
- Remove duplicate calls to System.unique_integer and extracted to single more contextual unique_id function.

### CODE REVIEW

File: lib/axon.ex [hunk 1]
```diff
@@ -1136,10 +1127,12 @@ defmodule Axon do
     end
   end
 
+  defp unique_id(modifiers) when is_list(modifiers), do: System.unique_integer(modifiers)
```
Comment (seanmor5) [inline lib/axon.ex:1130]: I don't think this is really all that necessary because it's an extra call to do the same thing without avoiding any duplication

**Review by seanmor5 — COMMENTED** [review]
I will fix the docs upstream, thanks!

**Comment by AlexJuca** [comment]
Go ahead @seanmor5

File: lib/axon.ex (hunk 1 above)
Comment (AlexJuca) [reply lib/axon.ex:1130]: Thanks, I'll try and add something with more value next time.

**Pull request closed without merging**